Makefile

```diff
@@ -1,5 +1,10 @@
 .PHONY: all test generate validate compile console dialyzer xref
 
+REBARVER = 3.13.2
+ifeq ($(OTPVER),24.0)
+	REBARVER = 3.15.1
```

Comment:
do we need it at all? Makefile uses local repo version of rebar3 anyway.

Reply:
I deleted that local version because there's no rebar version that compiles with both 24 and 20, so the script is downloading different rebars for different otp version. I followed the same pattern in most other repos.

Reply:
0-o-ops, GH shows diff for rebar3 like `Binary file not shown.`, I thought that it's just changed. now i see that we download it as makefile recipient.